src/extension/inspire/src/main/java/org/geoserver/inspire/ViewServicesUtils.java

```diff
@@ -32,19 +41,59 @@ public static void addScenario1Elements(
             translator.end("inspire_common:MediaType");
         }
         translator.end("inspire_common:MetadataUrl");
+        encodeSupportedLanguages(languages, translator);
+        translator.end("inspire_vs:ExtendedCapabilities");
+    }
+
+    private static String getDefaultLang(String languages, List<String> langList){
+        String defaultLang;
+        if (languages.contains(DEFAULT_SUFFIX))
+            defaultLang=langList.stream().filter(l->l.contains(DEFAULT_SUFFIX)).map(l->l.replace(DEFAULT_SUFFIX,"")).findFirst().get();
+        else
+            defaultLang="eng";
```

Comment:
Don't think this can be assumed, on upgrade the lone language configured previously should become the default one. Maybe the confiuration needs to keep the field for the language, and add a new one for the "other languages"

Reply:
fixed